### DISCUSSION TITLE
How to determine what profile update is happening

### DISCUSSION THREAD

**Discussion by stefanvanburen** [ory/kratos#1955] [Q&A] [answered]
Hi,

We have a form for users to update their email address, and a separate form for users to update their password. We have `privileged_session_max_age` set to a very low value, so that users effectively always have to re-enter their existing email and password via the sign in form.

When the sign in form is rendered, we make a `GetSelfServiceLoginFlow` call, which has `"forced": true` (we're on Kratos v0.5), which means that the login flow has been forced by the app.

Ideally, we'd be able to put some help text saying, "please sign in again to update your email", OR, "please sign in again to complete your password update". But it's not clear to me if it's possible to get that information from the API.

**Comment by aeneasr** [comment, marked as answer]
If you upgrade to Ory Kratos 0.7, this should resolve itself automatically as we have introduced this to the messages! :)
